applications/crossbar/priv/couchdb/account/devices.json

```diff
@@ -57,10 +63,43 @@
             ]
         },
         "listing_by_presence_id": {
-            "map": "function(doc) {if (doc.pvt_type != 'device' || doc.pvt_deleted) return;if (doc.provision && doc.provision.feature_keys) {for(var i in doc.provision.feature_keys) {var key = doc.provision.feature_keys[i];if(key.type == 'presence' || key.type == 'personal_parking') {emit(key.value, null);}}}}"
+            "map": [
+                "function(doc) {",
+                "  if (doc.pvt_type != 'device' || doc.pvt_deleted) return;",
+                "  if (doc.provision && doc.provision.feature_keys)",
+                "    for(var i in doc.provision.feature_keys) {",
+                "      var key = doc.provision.feature_keys[i];",
+                "      if(key.type == 'presence' || key.type == 'personal_parking')",
+                "        emit(key.value, null);",
+                "    }",
+                "}"
+            ]
         },
         "sip_credentials": {
-            "map": "function (doc) {if (typeof doc.sip !== 'undefined' && !doc.pvt_deleted) {var sip = JSON.parse(JSON.stringify(doc.sip)); sip.authorizing_id = doc._id; sip.authorizing_type = doc.pvt_type; emit(sip.username.toLowerCase(), sip);} else if (doc.pvt_type === 'sys_info' && !doc.pvt_deleted) {for (i in doc.servers) {if (!doc.servers[i].auth || !doc.servers[i].auth.auth_method || doc.servers[i].auth.auth_method.toLowerCase() != \"password\") continue; var auth = doc.servers[i].auth; var sip = {'password': auth.auth_password, 'username': auth.auth_user, 'method': auth.auth_method || 'password', 'invite_format': doc.servers[i].options.inbound_format || 'e164'}; sip.authorizing_id = doc._id; sip.authorizing_type = doc.pvt_type; emit(sip.username.toLowerCase(), sip);}}}"
+            "map": [
+                "function (doc) {",
+                "  if (typeof doc.sip !== 'undefined' && !doc.pvt_deleted) {",
+                "    var sip = JSON.parse(JSON.stringify(doc.sip));",
```

Comment:
I believe this was because the sip object wouldn't emit with the custom fields we were adding. It was introduced in 2011 though so this is my best guess :)